### PR TITLE
Update Day-O to version 2.0

### PR DESCRIPTION
**Pull request by ricardojoaoreis**
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/day-o.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@
   version :latest
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: Make this `version '2.0'`.

File: Casks/day-o.rb [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
   sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/day-o.rb [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
 
-  url 'http://shauninman.com/assets/downloads/Day-O.zip'
+  url 'http://shauninman.com/assets/downloads/Day-2.0.zip'
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: `url "http://shauninman.com/assets/downloads/Day-#{version}.zip"`

File: Casks/day-o.rb [hunk 4]
```diff
@@ -2,9 +2,9 @@
...
   name 'Day-O'
-  homepage 'http://www.shauninman.com/archive/2011/10/20/day_o_mac_menu_bar_clock'
+  homepage 'http://shauninman.com/archive/2016/10/20/day_o_2_mac_menu_bar_clock'
 
-  app 'Day-O/Day-O.app'
+  app 'Day-2.0/Day-O.app'
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: `app "Day-#{version}/Day-O.app"`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by patresi** [comment]
Implemented the changes you requested

File: Casks/day-o.rb [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
-  sha256 :no_check
+  version '2.0'
+  sha256 '24c64e66499bde5149d43c7b9b33b44b46ac6d136fcc4a988fd7c298aff02071'
 
   url 'http://shauninman.com/assets/downloads/Day-2.0.zip'
   name 'Day-O'
   homepage 'http://shauninman.com/archive/2016/10/20/day_o_2_mac_menu_bar_clock'
 
-  app 'Day-2.0/Day-O.app'
+  app 'Day-#{version}/Day-O.app'
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: These need to be double quotes, now, or the interpolation will not work.

File: Casks/day-o.rb [hunk 6]
```diff
@@ -1,10 +1,10 @@
 cask 'day-o' do
-  version :latest
-  sha256 :no_check
+  version '2.0'
+  sha256 '24c64e66499bde5149d43c7b9b33b44b46ac6d136fcc4a988fd7c298aff02071'
 
-  url 'http://shauninman.com/assets/downloads/Day-O.zip'
+  url 'http://shauninman.com/assets/downloads/Day-2.0.zip'
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: `url "http://shauninman.com/assets/downloads/Day-#{version}.zip"`

File: Casks/day-o.rb [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
   name 'Day-O'
-  homepage 'http://www.shauninman.com/archive/2011/10/20/day_o_mac_menu_bar_clock'
+  homepage 'http://shauninman.com/archive/2016/10/20/day_o_2_mac_menu_bar_clock'
 
-  app 'Day-O/Day-O.app'
+  app 'Day-#{version}/Day-O.app'
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: These need to be double quotes or the interpolation will not work.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
There are still changes missing. Made new comments.

File: Casks/day-o.rb [hunk 8]
```diff
@@ -1,10 +1,10 @@
 cask 'day-o' do
-  version :latest
-  sha256 :no_check
+  version '2.0'
+  sha256 '24c64e66499bde5149d43c7b9b33b44b46ac6d136fcc4a988fd7c298aff02071'
 
-  url 'http://shauninman.com/assets/downloads/Day-O.zip'
+  url 'http://shauninman.com/assets/downloads/Day-#{version}.zip'
```
Comment (vitorgalvao) [inline Casks/day-o.rb]: These also need to be double quotes. Same reason.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**